### PR TITLE
Opus release

### PR DESCRIPTION
**Pull request by svirpioj**
* set install to work with shared libs by default
* update GitHub workflows
* fix source package to include necessary files
* rename package to opus-fast-mosestokenizer

### CODE REVIEW

**Comment by svirpioj** [comment]
@jelmervdl, was your `github-release` branch I found ready? I haven't tested building the wheels yet. Let me also know if you notice something else that could be improved here.

File: .github/workflows/release-github.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: macos-wheels
+          path: dist
+
+      - name: Upload release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: dist/*.whl
```
Comment (jelmervdl) [inline .github/workflows/release-github.yaml:92]: Probably replace with:

```yaml
      - name: Publish wheels to PyPI
        env:
          TWINE_USERNAME: __token__
          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
        run: |
          python3 -m pip install twine
          twine upload *.whl
```

File: .github/workflows/release-github.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Build package and upload from docker
+      run: |
+        docker run --rm -v "${PWD}:/opt/fast-mosestokenizer" \
+          -e TWINE_USERNAME=__token__ \
+          -e TWINE_PASSWORD=${{ secrets.PYPI_TOKEN }} \
```
Comment (jelmervdl) [inline .github/workflows/release-github.yaml:62]: I don't think these lines are really necessary. That script doesn't upload anything anymore.

**Review by jelmervdl — COMMENTED** [review]
This branch for me only changed the uploading of the files to Github releases instead of PyPI, just as a workaround. I've only checked the wheel for macOS/Python3.9 and that seems okay.

Edit: different from the original branch, this builds Linux and Mac in parallel but uploads only once both builds are successful. I looked into building all versions in parallel ([like happens here](https://github.com/browsermt/bergamot-translator/blob/main/.github/workflows/build.yml#L24-L28)) but that seems to also include installing all dependencies for each version separately which seems like a bit of a waste.

**Pull request merged**